### PR TITLE
- Create a template for centos6.6

### PR DESCRIPTION
**Pull request by johnsonw**
- Create a template for centos6.6
- Create a base kickstart file for el6

Signed-off-by: Will Johnson <william.c1.johnson@gmail.com>

### CODE REVIEW

File: centos-lustre-templates/centos-6.6/centos66-1606-base-packerfile.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "_comment": "Packer description for a minimal CentOS Server",
+  "_comment": "Provides basis for creating Lustre servers and",
+  "_comment": "clients (compute nodes)",
```
Comment (jgrund) [inline centos-lustre-templates/centos-6.6/centos66-1606-base-packerfile.json]: I didn't address this in the other template, as it already existed; but I can't see how having multiple comment props on a JSON object is possible. The later should just overwrite the former.

File: centos-lustre-templates/centos-6.6/centos66-1606-base-packerfile.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      "iso_url":
+        "http://archive.kernel.org/centos-vault/6.6/isos/x86_64/CentOS-6.6-x86_64-bin-DVD1.iso",
+      "iso_checksum_url":
+        "http://archive.kernel.org/centos-vault/6.6/isos/x86_64/sha256sum.txt.asc",
+      "iso_checksum_type": "sha256",
+      "guest_additions_mode": "disable",
+      "headless": true,
+      "ssh_username": "vagrant",
+      "ssh_password": "vagrant",
+      "ssh_timeout": "15m",
```
Comment (jgrund) [inline centos-lustre-templates/centos-6.6/centos66-1606-base-packerfile.json]: What does this param reflect?

File: centos-lustre-templates/httpfiles/kickstart/base_el6.ks [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+reboot
+
+%packages --nobase --ignoremissing --excludedocs
+# vagrant needs this to copy initial files via scp
+openssh-clients
+# Prerequisites for installing VMware Tools or VirtualBox guest additions.
+# Put in kickstart to ensure first version installed is from install disk,
+# not latest from a mirror.
+kernel-headers
+kernel-devel
```
Comment (jgrund) [inline centos-lustre-templates/httpfiles/kickstart/base_el6.ks]: Are these the right headers / kernel for the lustre version you want to target?

File: centos-lustre-templates/centos-6.6/centos66-1606-base-packerfile.json (hunk 2 above)
Comment (johnsonw) [reply centos-lustre-templates/centos-6.6/centos66-1606-base-packerfile.json]: The ssh connection may timeout in certain cases (I think the default is 10 minutes) so I increased it to 15.

**Pull request merged**